projects/aca-testing-shared/src/pages/page.ts

```diff
@@ -82,13 +83,11 @@ export abstract class Page {
   }
 
   async isUploadDialogOpen(): Promise<boolean> {
-    return isPresentAndDisplayed(this.uploadDialog);
+    return this.uploadFilesDialog.isUploadDialogOpen();
   }
 
   async closeUploadDialog(): Promise<void> {
-    if (await this.isUploadDialogOpen()) {
-      await BrowserActions.click(this.closeUploadButton);
-    }
+    await this.uploadFilesDialog.closeUploadDialog();
```

Comment:
You can remove this one line method and call it directly in spec file

Reply:
closeUploadDialog() is also used in other spec file that I was not working on. Should I still remove it and change the other test file as well?

Reply:
Yes, please :)

Reply:
Done